lib/util/BlameGutter.js

```diff
@@ -187,8 +187,8 @@ export default class BlameGutter {
   }
 
   onResizeMove(e) {
-    const delta = e.pageX - this.resizeStartX;
-    this.updateGutterWidth(this.resizeStartWidth + delta);
+    const newWidth = this.resizeStartX ? e.pageX - this.resizeStartX + this.resizeStartWidth : this.width;
```

Comment:
i would write this a bit differently. no need to call `updateGutterWidth` at all if we dont have a `resizeStartX` right?

perhaps

```
if (!this.resizeStartX) {
  return;
}

// old code here
```